apps/OboeTester/app/src/main/java/com/mobileer/oboetester/TestDataPathsActivity.java

```diff
@@ -267,16 +280,25 @@ public String didTestFail() {
         StreamConfiguration requestedOutConfig = mAudioOutTester.requestedConfiguration;
         StreamConfiguration actualInConfig = mAudioInputTester.actualConfiguration;
         StreamConfiguration actualOutConfig = mAudioOutTester.actualConfiguration;
-        boolean passed = true;
         if (mMaxMagnitude <= MIN_REQUIRED_MAGNITUDE) {
             why += ", mag";
         }
-        if (mPhaseJitter > MAX_ALLOWED_JITTER) {
-            why += ", jitter";
+        if (!isPhaseJitterValid()) {
+            why += ", jitterUnknown";
+        } else if (getAveragePhaseError() > MAX_ALLOWED_JITTER) {
+            why += ", jitterHigh";
         }
         return why;
     }
 
+    private double getAveragePhaseError() {
+        return (mPhaseErrorCount > 0) ? (mPhaseErrorSum / mPhaseErrorCount) : MAX_ALLOWED_JITTER;
```

Comment:
Returning MAX_ALLOWED_JITTER if there is no error feels strange

Reply:
It is not that there is "no error". We just have no error measurements. I will bump up the value and comment it better.